nsv13/code/modules/overmap/weapons/damage.dm

```diff
@@ -85,7 +85,7 @@ Bullet reactions
 		addtimer(VARSET_CALLBACK(src, impact_sound_cooldown, FALSE), 1 SECONDS)
 	if(blocked)
 		return FALSE
-	if(CHECK_BITFIELD(overmap_deletion_traits, DAMAGE_STARTS_COUNTDOWN) && !(CHECK_BITFIELD(overmap_deletion_traits, DAMAGE_DELETES_UNOCCUPIED) && !has_occupants())) //Code for handling "superstructure crit" countdown
+	if(CHECK_BITFIELD(overmap_deletion_traits, DAMAGE_STARTS_COUNTDOWN) && !(CHECK_BITFIELD(overmap_deletion_traits, DAMAGE_DELETES_UNOCCUPIED) && !has_occupants()) && !ghost_controlled) //Code for handling "superstructure crit" countdown
```

Comment:
Setting damage_deletion_traits to DAMAGE_ALWAYS_DELETES (a.k.a 0) when a ghost takes possession of a ship will fix this better

Reply:
Done